delta/ml/train.py

```diff
@@ -199,14 +214,17 @@ def train(model_fn, dataset : ImageryDataset, training_spec):
     if config.mlflow.enabled():
         mcb = _mlflow_train_setup(model, dataset, training_spec)
         callbacks.append(mcb)
+        print('Using mlflow folder: ' + mlflow.get_artifact_uri())
```

Comment:
delete (or verbose mode)

Reply:
Is there somewhere else the mlflow folder is printed to?  I need that information to record the results from processing runs.

Reply:
What if we just add a verbose flag, then do if config.verbose() and you can use -v or change the config flag? I expect most users will run the web interface and not need this.